action.yml

```diff
@@ -5,15 +5,87 @@ branding:
   icon: tag
   color: yellow
 inputs:
-  gh-token:
-    description: GitHub token with repo access
-    required: true
+  push-tag:
+    description: 'Push the tag to the origin repository'
+    required: false
+    default: true # TODO: default false in v1?
+  push-major-version-branch:
+    description: 'Push to a branch matching the major version number on the origin repository'
+    required: false
+    default: true # TODO: default false in v1?
+  v-prefix:
+    description: "Whether to prefix the tag with the letter 'v'."
+    required: false
+    default: true # TODO: default false in v1?
+  create-release:
+    description: "Whether to create a release from the tag or not. 'true', 'false'." # TODO: Support 'draft'
+    required: false
+    default: 'true' # TODO: default false in v1?
```

Comment:
why in quotes?
```suggestion
    default: true # TODO: default false in v1?
```

Reply:
its a string either way but good to be consistent